scrapenews/middlewares.py

```diff
@@ -41,7 +41,7 @@ def process_spider_exception(self, response, exception, spider):
 
         # Should return either None or an iterable of Response, dict
         # or Item objects.
-        pass
+        return None
```

Comment:
Why are you changing this? Is it a flake8 thing? A python 3 thing? This was copied from scrapy source and wasn't a problem AFAICT

Reply:
The other parts of the file `return None` explicitly. So I wanted it to match those. But I see the two exception methods use pass and that is the pattern. I'll turn if back